esper/__init__.py

```diff
@@ -174,6 +174,22 @@ def get_processor(self, processor_type: _Type[_P]) -> _Optional[_P]:
         else:
             return None
 
+    def create_known_entity(self, entity: int):
+        """Create a new entity using known id
+
+        Raises a ValueError if an entity as already been created using this ID.
+        Empty and dead entities will count as created ones.
+
+        :param entity: Entity ID
+        :return Entity ID
+        """
+        if entity in self._entities:
+            raise ValueError("Entity ID %i already exists" % entity)
+        self._entities[entity] = {}
+        if entity > self._next_entity_id:
```

Comment:
I'm not sure about that, so if you inject nr 100, counter is moved there.
Might be that incrementing counter should be changed to omit existing entities?